src/components/infoWindow.vue

```diff
@@ -80,11 +172,12 @@ export default {
     if (this.$infoWindow) {
       this.$infoWindow.setMap(null);
     }
+    this.$off('marker-ready',this.markerReady);
```

Comment:
i'm not sure if you really need this. The component is going to be destroyed, so I'm pretty sure the events aren't going to be fire any more.